text/0000-support-insecure-registries.md

```diff
@@ -0,0 +1,72 @@
+# Meta
+[meta]: #meta
+- Name: Support for pull images from or push images to insecure image registries.
+- Start Date: 2022-08-17
+- Author(s): wanjunlei
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This RFC describes how to pull images form or push images to insecure image registries when build images using buildpacks.
+
+# Definitions
+[definitions]: #definitions
+
+Insecure image registry - The insecure image registry mentioned in this RFC refers to the image registry using the http protocol, and with a non-internal ip or a domain name. Because currently buildpacks can access inecure image registries using internal ip.
```

Comment:
```suggestion
Insecure image registry - The insecure image registry mentioned in this RFC refers to the image registry using the http protocol, and with a non-internal ip or a domain name. Because currently buildpacks can access insecure image registries using internal ip.
```